pkg/syncer/syncer.go

```diff
@@ -83,13 +86,23 @@ func StartSyncer(ctx context.Context, cfg *SyncerConfig, numSyncerThreads int, i
 	logger.V(2).Info("starting syncer")
 
 	kcpVersion := version.Get().GitVersion
```

Comment:
This is not the `kcp` version, but the compiled version of the `syncer`, right? Are you attempting to add a check in this PR that the compiled version of the `syncer` matches the compiled version of the `kubectl kcp workload sync` command? The comments & code read like you're checking the `kcp` server version, which I don't think is what this is doing?

Reply:
> Are you attempting to add a check in this PR that the compiled version of the syncer matches the compiled version of the kubectl kcp workload sync command
That's precisely what I'm doing yes, afaict.

> The comments & code read like you're checking the kcp server version

Can you point me to the comments you find confusing in this regard ?

Reply:
The variable in the code line that this comment is on is called `kcpVersion` - it's not the version of `kcp`. It's the version of the `syncer` :)

Reply:
This variable was named this way previously, since the Syncer used to be compiled with KCP I assume. But I can rename it to make it clearer in this context.

Reply:
Done